packages/ui/src/components/DataMapper/debug/vite.config.js

```diff
@@ -15,5 +15,15 @@ export default defineConfig(async () => {
         },
       },
     },
+    resolve: {
+      alias: [
+        {
+          find: /^~.+/,
+          replacement: (val) => {
```

Comment:
we're using `vite` also in storybook, so it would probably be convenient to normalize these configs.